build.cake

```diff
@@ -197,31 +197,16 @@ Task("NuGet")
     MoveFiles("Microsoft.AppCenter*.nupkg", "output");
 }).OnError(HandleError);
 
-// Replace version placeholder in nuspecs
-Task("PrepareNuspecsForVSTS").Does(()=>
-{
-    foreach (var module in AppCenterModules)
-    {
-        ReplaceTextInFiles(System.IO.Path.Combine(NuspecFolder, module.MainNuspecFilename), "$version$", module.NuGetVersion);
-    }
-});
-
-Task("PrepareAssemblyPathsVSTS").Does(()=>
+Task("NugetPackAzDO").Does(()=>
 {
     var nuspecPathPrefix = EnvironmentVariable("NUSPEC_PATH");
     foreach (var module in AppCenterModules)
     {
         var nuspecPath = System.IO.Path.Combine(nuspecPathPrefix, module.MainNuspecFilename);
+        ReplaceTextInFiles(nuspecPath, "$version$", module.NuGetVersion);
```

Comment:
Curious why we need that line since we call set release version already in the build pipeline and it does more.

Reply:
set release version modifies xml config, it doesn't touch nuspecs